templates/stats.ejs

```diff
@@ -1,179 +1,236 @@
-    onclick="updateStat('toughness', 'v_toughness_aux', 'toughness', 'toughness_value')"
-    style="grid-column: 1; grid-row: 4;">
-        Toughness: 
+    <button class="stat-field" id="toughness"
+            onclick="updateStat('toughness', 'v_toughness_aux', 'toughness', 'toughness_value')"
+            style="grid-column: 1; grid-row: 4;">
+        Toughness:
         <span id="toughness_value">
-        <% if (locals.v_toughness) { %> <%= v_toughness %> <% } else {%> 0 <% } %>
+            <% if (locals.v_toughness) { %> <%= v_toughness %>
+            <% } else { %> 0
+            <% } %>
         </span>
         /
-        <% if (locals.v_max_stat) { %> <%= v_max_stat %> <% } else {%> 3 <% } %>
+        <% if (locals.v_max_stat) { %> <%= v_max_stat %>
+        <% } else { %> 3
+        <% } %>
+    </button>
 
-    <button class="stat-field" id="agility" 
-    onclick="updateStat('agility', 'v_agility_aux', 'agility', 'agility_value')"
-    style="grid-column: 2; grid-row: 3;">
-        Speed: 
+    <button class="stat-field" id="agility"
+            onclick="updateStat('agility', 'v_agility_aux', 'agility', 'agility_value')"
+            style="grid-column: 2; grid-row: 3;">
+        Speed:
         <span id="agility_value">
-        <% if (locals.v_agility) { %> <%= v_agility %> <% } else {%> 0 <% } %>
+            <% if (locals.v_agility) { %> <%= v_agility %>
+            <% } else { %> 0
+            <% } %>
         </span>
         /
-        <% if (locals.v_max_stat) { %> <%= v_max_stat %> <% } else {%> 3 <% } %>
+        <% if (locals.v_max_stat) { %> <%= v_max_stat %>
+        <% } else { %> 3
+        <% } %>
     </button>
 
     <button class="stat-field" id="reflexes"
-    onclick="updateStat('reflexes', 'v_reflexes_aux', 'reflexes', 'reflexes_value')"
-    style="grid-column: 2; grid-row: 4;">
-        Reflexes: 
+            onclick="updateStat('reflexes', 'v_reflexes_aux', 'reflexes', 'reflexes_value')"
+            style="grid-column: 2; grid-row: 4;">
+        Reflexes:
         <span id="reflexes_value">
-        <% if (locals.v_reflexes) { %> <%= v_reflexes %> <% } else {%> 0 <% } %>
+            <% if (locals.v_reflexes) { %> <%= v_reflexes %>
+            <% } else { %> 0
+            <% } %>
         </span>
         /
-        <% if (locals.v_max_stat) { %> <%= v_max_stat %> <% } else {%> 3 <% } %>
+        <% if (locals.v_max_stat) { %> <%= v_max_stat %>
+        <% } else { %> 3
+        <% } %>
     </button>
 
     <button class="stat-field" id="hearing"
-    onclick="updateStat('hearing', 'v_hearing_aux', 'hearing', 'hearing_value')"
-    style="grid-column: 1; grid-row: 5;">
+            onclick="updateStat('hearing', 'v_hearing_aux', 'hearing', 'hearing_value')"
+            style="grid-column: 1; grid-row: 5;">
         Hearing:
         <span id="hearing_value">
-        <% if (locals.v_hearing) { %> <%= v_hearing %> <% } else {%> 0 <% } %>
+            <% if (locals.v_hearing) { %> <%= v_hearing %>
```

Comment:
This style is arguably less readable than the previous one. Also all of these could be collapsed to something like
```suggestion
             <%= locals.v_hearing || 0 %>
```
(I think)